collections/reports/labelmanager.php

```diff
@@ -460,27 +451,14 @@ function labelFormatChanged(selObj){
 										<div style="margin:10px">
 											<input type="submit" name="submitaction" onclick="return changeFormExport(this,'labeldynamic.php','_self');" value="<?php echo (isset($LANG['EXP_CSV']) ? $LANG['EXP_CSV'] : 'Export to CSV') ?>" />
 										</div>
-										<?php
-										if($reportsWritable){
-											?>
-											<div style="margin:10px">
-												<input type="submit" name="submitaction" onclick="return changeFormExport(this,'labelsword.php','_self');" value="<?php echo (isset($LANG['EXP_DOCX']) ? $LANG['EXP_DOCX'] : 'Export to DOCX') ?>" />
-											</div>
-											<?php
-										}
-										?>
-									</div>
-										<?php
-										if($reportsWritable){
-											?>
-											<div style="clear:both;padding:10px 0px">
-												<b><?php echo (isset($LANG['NOTE']) ? $LANG['NOTE'] : 'Note:') ?></b> <?php echo (isset($LANG['NOTE_1']) ? $LANG['NOTE_1'] : 'Currently, Word (DOCX) output only generates the old static label format.') ?><br/><?php echo (isset($LANG['NOTE_2']) ? $LANG['NOTE_2'] : 'Output of variable Label Formats (pulldown options) as a Word document is not yet supported.') ?><br/>
-												<?php echo (isset($LANG['NOTE_3']) ? $LANG['NOTE_3'] : 'A possible work around is to print labels as PDF and then convert to a Word doc using Adobe tools.') ?><br/>
-												<?php echo (isset($LANG['NOTE_4']) ? $LANG['NOTE_4'] : 'Another alternatively, is to output the data as CSV and then setup a Mail Merge Word document.') ?>
-											</div>
-											<?php
-										}
-										?>
+										<div style="margin:10px">
+											<input type="submit" name="submitaction" onclick="return changeFormExport(this,'labelsword.php','_self');" value="<?php echo (isset($LANG['EXP_DOCX']) ? $LANG['EXP_DOCX'] : 'Export to DOCX') ?>" />
+										</div>
+										<div style="clear:both;padding:10px 0px">
+											<b><?php echo (isset($LANG['NOTE']) ? $LANG['NOTE'] : 'Note:') ?></b> <?php echo (isset($LANG['NOTE_1']) ? $LANG['NOTE_1'] : 'Currently, Word (DOCX) output only generates the old static label format.') ?><br/><?php echo (isset($LANG['NOTE_2']) ? $LANG['NOTE_2'] : 'Output of variable Label Formats (pulldown options) as a Word document is not yet supported.') ?><br/>
+											<?php echo (isset($LANG['NOTE_3']) ? $LANG['NOTE_3'] : 'A possible work around is to print labels as PDF and then convert to a Word doc using Adobe tools.') ?><br/>
+											<?php echo (isset($LANG['NOTE_4']) ? $LANG['NOTE_4'] : 'Another alternatively, is to output the data as CSV and then setup a Mail Merge Word document.') ?>
```

Comment:
Typo here. "Another alternatively," should change to "Another alternative", and the comma should be removed.